src/core/lib/iomgr/timer_generic.cc

```diff
@@ -223,7 +223,7 @@ static void validate_non_pending_timer(grpc_timer* t) {
  * has last-seen. This is an optimization to prevent the thread from checking
```

Comment:
the comment above this block is slightly out of date now; let me know if you'd like me to update it. The incorrect part is "intptr_t" -- the real limitation is that TLS types must not be larger than a machine word.

Reply:
I don't think it matters terribly - we'll nuke this code in a few months.